superdesk/publish/formatters/ninjs_formatter.py

```diff
@@ -200,6 +200,18 @@ def _transform_to_ninjs(self, article, subscriber, recursive=True):
         if article.get('attachments'):
             ninjs['attachments'] = self._format_attachments(article)
 
+        if 'body_html' in ninjs or 'body_text' in ninjs:
+            if 'body_html' in ninjs:
+                word_count = etree.get_word_count(ninjs.get('body_html'))
+            else:
+                word_count = etree.get_text_word_count(ninjs['body_text'])
+            reading_time_float = word_count / 250
```

Comment:
imo this should go into a function with some docs explaining the formula

Reply:
@petrjasek : OK, do you know about a module where I can put that ? Etree seems not adapted since it's only from word count (or I can add get_reading_time in etree, but it would need to count the word several times if word count is also needed).

Reply:
right, not sure why we have `get_word/text_count` in etree anyhow, so imo all those should go into textutils or something